src/scripts/food/food_filterdata.js

```diff
@@ -6,6 +6,7 @@ const foodFilter = (foodObj) => {
 
 // Loop through my array and collect the name and location of each restaurant. T
 const foodArrayLoop = (newArray) => {
+  buildClear();

```

Comment:
Will we all need to add this to our filter.js?  Or can this be added as the first function in our search button event listeners? Just wondering if there's a place to make this DRY-er

Reply:
... I see now that you DID add it to our files. Haaaaaaa nevermind.